src/content/snippets/styled-components/Header.js.mdx

```diff
@@ -0,0 +1,74 @@
+```js
+// ./src/stories/Header.js
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { Button } from './Button';
+import './header.css';
```

Comment:
Still used?

Reply:
Yes, just to avoid needing to rewrite the entire component with the same styles